homeassistant/components/imeon_inverter/sensor.py

```diff
@@ -0,0 +1,449 @@
+"""Imeon inverter sensor support."""
+
+import logging
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    UnitOfElectricCurrent,
+    UnitOfElectricPotential,
+    UnitOfEnergy,
+    UnitOfFrequency,
+    UnitOfPower,
+    UnitOfTemperature,
+    UnitOfTime,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
```

Comment:
```suggestion
from homeassistant.helpers.entity_platform import AddConfigEntryEntitiesCallback
```
has changed in the meantime, might need a rebase